gvsbuild/patches/libsass/LICENSE.build

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2021 The Meson development team
```

Comment:
meson development team?

Reply:
Hey @nacho, maybe I am missing the point -
libsass has a LICENSE file that is being installed during the build of the project. This is a LICENSE.build file for where we got the build wrapdb from.